tools/syntax.txt

```diff
@@ -1096,6 +1110,40 @@ LetPortListSyntax portList
 EqualsValueClauseSyntax initializer
 token semi
 
+AssertionVariableDeclarationSyntax kind=AssertionVariableDeclaration
```

Comment:
I think you should be able to reuse DataDeclarationSyntax here; they have the same meaning and layout.